recipes/cctag/all/conanfile.py

```diff
@@ -129,7 +142,7 @@ def package(self):
         copy(self, "COPYING.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
-        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        # rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
Hi @valgur  this commented removal of CCTagConfig.cmake make pipeline failed because of KB-H016 hook.